tests/test_async_processing_import_export.py

```diff
@@ -323,6 +404,46 @@ def test_sentinel_import_error(self):
         self.waitAsyncStatusAssertHTTP(rv, headers=self.admin_auth_header,
                                        http_status=400, status="error")
 
+    # Test for STAC
+    @unittest.skipIf('GOOGLE_APPLICATION_CREDENTIALS' not in os.environ and 'GOOGLE_CLOUD_PROJECT' not in os.environ,
+                     "Test is skipped because 'GOOGLE_APPLICATION_CREDENTIALS' and 'GOOGLE_CLOUD_PROJECT' not set")
+    def test_stac_import(self):
+
```

Comment:
Could you please add a sentence, saying what this test is doing? We agreed upon starting this for all new tests, so no great example until now, but then we have a good example ;)